modin/backends/pandas/parsers.py

```diff
@@ -153,8 +153,11 @@ def get_dtypes(cls, dtypes_ids):
             pandas.Series where index is columns names and values are
             columns dtypes.
         """
+        partitions_dtypes = cls.materialize(dtypes_ids)
+        if all([len(dtype) == 0 for dtype in partitions_dtypes]):
```

Comment:
Can you elaborate why this change fixes the issue?

Reply:
Yes, this change is needed because when empty frame should be read, each worker will read empty frame and it's `dtypes` will be empty `Series` that corresponds to `dtypes=None`. If we skip this check, `dtypes` will values will be calculated incorrectly because `partitions_dtypes` contains no columns.